TESTING.md

```diff
@@ -0,0 +1,74 @@
+Artisan uses many libraries, and it is important that they function correctly.
+Some packages have built-in tests that can be run.
+
+Note that this is especially important to do when building OpenBLAS, NumPy and/or SciPy from source,
+and upgrading one of them, as they strongly depend on each other, and may not always be compatible
+(e.g. [scipy#19831](https://github.com/scipy/scipy/issues/19831))
+
+The Flatpak does by default not include everything to run these tests, so you'll
+need to build an adapted Flatpak to run them.
+
+## numpy & scipy
```

Comment:
As a sidenote: I did extensive testing on NumPy and Scipy, and only got their tests to work if I used the wheels, as much as I wanted to build from source.